src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -223,6 +223,45 @@ void ICACHE_RAM_ATTR SX127xDriver::SetFrequencyReg(uint32_t freq)
   hal.writeRegisterBurst(SX127X_REG_FRF_MSB, outbuff, sizeof(outbuff));
 }
 
+void ICACHE_RAM_ATTR SX127xDriver::SetRxTimeoutUs(uint32_t interval)
+{
+  timeoutSymbols = 0; // no timeout i.e. use continuous mode
+  if (interval)
+  {
+    int spread = 0;
+    switch (currSF)
+    {
+    case SX127x_SF_6:
+      spread = 6;
+      break;
+    case SX127x_SF_7:
+      spread = 7;
+      break;
+    case SX127x_SF_8:
+      spread = 8;
+      break;
+    case SX127x_SF_9:
+      spread = 9;
+      break;
+    case SX127x_SF_10:
+      spread = 10;
+      break;
+    case SX127x_SF_11:
+      spread = 11;
+      break;
+    case SX127x_SF_12:
+      spread = 12;
+      break;
+    }
+    double symbolTimeUs = ((double)(1 << spread)) / GetCurrBandwidth() * 1000000;
+    timeoutSymbols = interval / symbolTimeUs;
+    hal.setRegValue(SX127X_REG_SYMB_TIMEOUT_LSB, timeoutSymbols);
+    DBGLN("Interval: %d", interval);
+    DBGLN("symbolTimeUs: %d", (uint32_t)symbolTimeUs);
+    DBGLN("Timeout: %d symbols", timeoutSymbols);
```

Comment:
Three debug lines? This ain't Serial.print! :-D put them all together then comment it out cause this is not useful debug info beyond getting it working.
`DBGLN("SetRxTimeout(%u), symbolTime=%uus symbols=%u", interval, (uint32_t)symbolTimeUs, timeoutSymbols)`

Also
```
unsigned int spread;
uint32_t symbolTimeUs =(1 << spread) * 100000 / GetCurrBandwidth();
timeoutSymbols = interval / symbolTimeUs;
```

Reply:
Sorry, was my fault

Reply:
> ```
> unsigned int spread;
> uint32_t symbolTimeUs =(1 << spread) * 100000 / GetCurrBandwidth();
> timeoutSymbols = interval / symbolTimeUs;
> ```

We'd have to use uint64_t because 1<<12 * 1000000 (you are missing a 0) is bigger than uint32!

Reply:
Nope, I'm wrong! Damn you math!